Editor/Editor.vb

```diff
@@ -1021,6 +1028,14 @@ Public Class Editor
         Reload()
     End Sub
 
+    Private Sub m_fileWatcherQuest_Changed(sender As Object, e As System.IO.FileSystemEventArgs) Handles m_fileWatcherQuest.Changed

```

Comment:
This duplicates the regular file watcher, so you could use the same event handler function here (though I would rename it). Although, if you were monitoring the app directory, you would want the banner to say that (right now it will tell you that the game's `.aslx` file has changed, when it hasn't).

Reply:
My knowledge is not sufficient for this